engine/mpm_solver.py

```diff
@@ -130,6 +130,7 @@ def __init__(
             self.material = ti.field(dtype=ti.i32)
         # Particle color
         self.color = ti.field(dtype=ti.i32)
+        self.color_with_alpha = ti.Vector.field(4, dtype=ti.f32)
```

Comment:
Shall we add a `ggui_support=False` flag so that this field is allocated only if we use GGUI?.